sdk/spring/spring-cloud-azure-feature-management/src/test/java/com/azure/spring/cloud/feature/management/filters/recurrence/RecurrenceEvaluatorTest.java

```diff
@@ -0,0 +1,534 @@
+        range4.setNumberOfRecurrences(2);
+        recurrence4.setRange(range4);
+        recurrence4.setPattern(pattern4);
+        settings4.setStart("2023-09-01T00:00:00+08:00");
+        settings4.setEnd("2023-09-01T00:00:01+08:00");
+        settings4.setRecurrence(recurrence4);
+        consumeEvaluationTestData(settings4, now4, false);
+    }
+
+    @Test
+    public void dailyEndDateFalse() {
+        final ZonedDateTime now = ZonedDateTime.parse("2023-09-04T00:00:00+08:00");
+        final TimeWindowFilterSettings settings = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern = new RecurrencePattern();
+        final RecurrenceRange range = new RecurrenceRange();
+        final Recurrence recurrence = new Recurrence();
+        pattern.setType("Daily");
+        range.setType("EndDate");
+        range.setEndDate("2023-09-03T00:00:00+08:00");
+        recurrence.setRange(range);
+        recurrence.setPattern(pattern);
+        settings.setStart("2023-09-01T00:00:00+08:00");
+        settings.setEnd("2023-09-01T00:00:01+08:00");
+        settings.setRecurrence(recurrence);
+        consumeEvaluationTestData(settings, now, false);
+    }
+
+    @Test
+    public void dailyDiffTimeZoneTrue() {
+        final ZonedDateTime now = ZonedDateTime.parse("2023-09-02T16:00:00+00:00"); // 2023-09-03T00:00:00+08:00
+        final TimeWindowFilterSettings settings = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern = new RecurrencePattern();
+        final RecurrenceRange range = new RecurrenceRange();
+        final Recurrence recurrence = new Recurrence();
+        pattern.setType("Daily");
+        pattern.setInterval(2);
+        recurrence.setRange(range);
+        recurrence.setPattern(pattern);
+        settings.setStart("2023-09-01T00:00:00+08:00");
+        settings.setEnd("2023-09-01T00:12:01+08:00");
+        settings.setRecurrence(recurrence);
+        consumeEvaluationTestData(settings, now, true);
+    }
+
+    @Test
+    public void dailyDiffTimeZoneFalse() {
+        final ZonedDateTime now = ZonedDateTime.parse("2023-09-02T15:59:00+00:00"); // 2023-09-02T23:59:00+08:00
+        final TimeWindowFilterSettings settings = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern = new RecurrencePattern();
+        final RecurrenceRange range = new RecurrenceRange();
+        final Recurrence recurrence = new Recurrence();
+        pattern.setType("Daily");
+        pattern.setInterval(2);
+        recurrence.setRange(range);
+        recurrence.setPattern(pattern);
+        settings.setStart("2023-09-01T00:00:00+08:00");
+        settings.setEnd("2023-09-01T00:12:01+08:00");
+        settings.setRecurrence(recurrence);
+        consumeEvaluationTestData(settings, now, false);
+    }
+
+    @Test
+    public void weeklyDaysOfWeekTrue() {
+        final ZonedDateTime now1 = ZonedDateTime.parse("2023-09-04T00:00:00+08:00");    // Monday in the 2nd week
```

Comment:
Could you adjust the number of whitespaces before the comments to make it consistent.

Reply:
The same for the else comments

Reply:
Sure. I used to use `tab` instead of `space`. so the whilte spaces maybe different. Updated to use `space`